ld.so.conf

```diff
@@ -1 +1,4 @@
+include /run/flatpak/ld.so.conf.d/app-*-org.freedesktop.Platform.GL32.*.conf
```

Comment:
Should these things just work in the runtime by default?

Reply:
Flatpaked Steam says "We just make any GL32 extension have higher priority" to that line. Oversaw to copy the comment too. I can't say if it a "just in case" thing or outdated.

Reply:
It might be worth discussing with the fdo folks, *shrug*.

Thanks for tracking it down.